wiki/People/Tournament_Committee/en.md

```diff
@@ -97,6 +97,8 @@ New members to the Tournament Committee must meet the following basic criteria b
 
 Whether a prospective member applies via email or is brought forward by an existing member, the process is the same. The Tournament Committee holds an in-depth discussion on the candidate. Special attention is paid towards their activity in the tournament scene, stature, community contributions, and how they would benefit the committee. Following said discussion, an anonymous vote is held to determine whether to add the candidate to the committee.
 
+Users that fail to gather enough positive votes will be denied, and won't be able to apply for a period of 6 months from the moment they receive a response to their application which will be delivered no later than 2 weeks from the day they apply.
```

Comment:
I'm assuming this part only goes for people who actually *applied*, right? the previous paragraph makes it sound like this whole section is about both applicants and ppl who were just suggested by existing staff

Reply:
In both cases, both would have the same cooldowns plus app response times. The only thing that would potentially change is the way the additions are communicated. Otherwise, the internal processes are all the same.

Reply:
so it's possible that I could apply for the first time and be told "sorry, still have 3 months of your cooldown left from the last time we brought you up", even though that wasn't my decision? that doesn't sound right to me but if it's actually how it is then okay.

Reply:
Will have to ask TComm members about the nuances of it. By the internal docs, I do not see a concrete interpretation on whether the voting is with the user's previous consent, if it's held and then the user is asked to be added afterwards, or what. Forwarding.

Reply:
For now, let's change this to:

> Applicants that fail to gather enough positive votes will be denied and will not be able to reapply for a period of 6 months. This cooldown begins the moment an applicant receives a response to their application. This notification will be delivered no later than 1 week from the date of their application.

Reply:
Applicants can now **only** apply via email. This has been made clear. If any member in the team wants to bring up a candidate, they should still apply via email. No exceptions. Under this premise, the cooldown on failed applications will always apply.

An [internal vote](https://pif2.ephemeral.ink/discussions/214) was also held. The 6 month cooldown will stay as-is.